pkg/blobstore/configuration/new_blob_access.go

```diff
@@ -121,7 +122,14 @@ func NewNestedBlobAccess(configuration *pb.BlobAccessConfiguration, creator Blob
 			if err != nil {
 				return nil, err
 			}
-			implementation = blobstore.NewCloudBlobAccess(bucket, backend.Cloud.KeyPrefix, storageType)
+			var refreshInterval time.Duration
+			if backendConfig.S3.RefreshInterval != nil {
```

Comment:
Trailing comment after I already pressed the submit button:

What are the costs (in terms of money) of sending COPY requests? If this is more expensive than plain HEAD requests, should we make this option required? I just want to make sure that existing users of CloudBlobAccess don't run into unintended expenses by upgrading to this new code.

Maybe this should also be documented.

Reply:
S3 bills this as if it was a PUT request: https://aws.amazon.com/s3/pricing/

That is indeed more expensive, and making it optional seems reasonable. The cost is likely inconsequential.

Reply:
Documented in the proto comments, and now it's a required field because that seems reasonable.

The way to "disable" it is obvious after a little thought: just set this to any value larger than your expiration time. Or just any large value.